<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerTimeSchedule</class>
 <widget class="QWidget" name="LayerTimeSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="wBgTime" native="true">
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="spacing">
       <number>40</number>
      </property>
      <item row="3" column="2">
       <widget class="QTimeEdit" name="teFrom">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbTimeDayOfWeek">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>星期一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期二</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期三</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期四</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期五</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期六</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期日</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QCheckBox" name="cbStopPreVideo">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>切換頁面中止先前影片播放</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QTimeEdit" name="teTo">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="lbLayerNameForTime">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>樣板</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>在裝置設定打開啟用</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="2">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
